tasks/install.yml

```diff
@@ -29,15 +29,25 @@
     owner: "{{ pyenv_owner }}"
     mode: "0644"
 
-- name: "Load pyenv env variables in {{ pyenv_setting_path }}"
-  ansible.builtin.lineinfile: dest="{{ pyenv_setting_path }}"
+- name: Use legacy variable pyenv_setting_path if available
+  ansible.builtin.set_fact:
+    pyenv_shellrc_file: "{{ pyenv_setting_path }}"
+  when: pyenv_setting_path is defined
+
+- name: Print deprecation warning if pyenv_setting_path defined
+  ansible.builtin.debug:
+    msg: pyenv_setting_path is deprecated, use pyenv_shellrc_file instead
+  when: pyenv_setting_path is defined
+
+- name: "Load pyenv env variables in {{ pyenv_shellrc_file }}"
+  ansible.builtin.lineinfile: dest="{{ pyenv_shellrc_file }}"
     regexp="\.pyenvrc$"
     line="source {{ pyenvrc_path }}/.pyenvrc"
     state=present
     create=yes
 
-- name: "Add pyenv autocomplete in {{ pyenv_setting_path }}"
-  ansible.builtin.lineinfile: dest="{{ pyenv_setting_path }}"
+- name: "Add pyenv autocomplete in {{ pyenv_shellrc_file }}"
+  ansible.builtin.lineinfile: dest="{{ pyenv_shellrc_file }}"
```

Comment:
Another small improvement for consistency is to not use free-form parameters for actions, also ansible-lint favors thuis sintax.

Instead of:
```
  ansible.builtin.lineinfile: dest="{{ pyenv_shellrc_file }}"
...
```

Use:
```
  ansible.builtin.lineinfile:
    dest: "{{ pyenv_shellrc_file }}"
...
```

Reply:
@staticdev This is not a row that I have  added,  only changed the name of the parameter. OK with any changes/improvements you would like to do in your code.

Reply:
@staticdev in the lint PR (#161 ) I fixed the above and all other lint warnings. When that one is merged I can rebase this.